doc/crds.md

```diff
@@ -251,6 +251,8 @@ Where:
       properly.
 	* **NOTAPPLICABLE**: Which indicates that the check didn't run because it is not
       applicable or not selected.
+ * **valueUsed**: a list of settable varaibles associated with the rule scan result,
+  a user can set these varaibles in a tailord profile.
```

Comment:
s/valuesed/valuesUsed/
s/varaibles/variables/g
s/tailord/tailored/

Reply:
Haha thanks for pointing out the typo